src/libraries/System.Runtime.Extensions/tests/System/EnvironmentTests.cs

```diff
@@ -393,7 +396,7 @@ public void GetSystemDirectory()
         [Theory]
         [PlatformSpecific(TestPlatforms.AnyUnix)]  // Tests OS-specific environment
         [InlineData(Environment.SpecialFolder.UserProfile, Environment.SpecialFolderOption.None)]
-        [InlineData(Environment.SpecialFolder.MyDocuments, Environment.SpecialFolderOption.None)] // MyDocuments == Personal
+        [InlineData(Environment.SpecialFolder.MyDocuments, Environment.SpecialFolderOption.DoNotVerify)] // MyDocuments == Personal
```

Comment:
Why the change to `DoNotVerify` here?

Reply:
From what I remember, it was because the documents folder along with pictures/videos/music folder etc. don't exist on the test environment and thus would fail when being tested against.

Reply:
is that only for certain platforms? Which platforms does this folder not exist?

Reply:
I think only for Linux.
When I made the PR initially without that change, I couldn't run the tests locally, so I relied on the CI here to check whether tests succeed or not. They at first failed due to the folders not existing, so I changed it to DoNotVerify to be consistent with the other "media" folders and make the tests pass. It has been a few months, so don't remember all the details. I force pushed a lot, so comparing the commits is difficult + the original helix logs have become invalid now.